cmd/podman/common/create.go

```diff
@@ -449,6 +449,11 @@ func GetCreateFlags(cf *ContainerCLIOpts) *pflag.FlagSet {
 		"tty", "t", false,
 		"Allocate a pseudo-TTY for container",
 	)
+	createFlags.StringVar(
+		&cf.TZ,
+		"tz", containerConfig.TZ(),
```

Comment:
What does this do in podman-remote?

Reply:
ah, I haven't done anything in the remote area, should I add the remote implementation to this PR as well?

Reply:
if you don't do it here, i would exect you to wire up the tz stuff asap for the api (remote) side.

Reply:
I can definitely do that, but it might take a day or so to dig around and understand the api side. Would you prefer it in a new pr or add to this one?

Reply:
@baude uhh it looks like somehow i actually got this wired to the api remote side without knowing i did. pleasant surprise(?)